canvas_modules/common-canvas/package.json

```diff
@@ -20,30 +20,30 @@
     "debug": "NODE_ENV=test node --inspect-brk node_modules/.bin/jest --runInBand"
   },
   "dependencies": {
-    "@carbon/icons-react": "^10.13.0",
```

Comment:
These had to be set back to static version since there was a breaking change in the newer carbon builds.  Since we ship a commonjs module all modules are includes so allowing for moving up to newer modules doesn't help.  If we move to a `module` type distribution we should revisit this.